app/Post.php

```diff
@@ -6,5 +6,22 @@
 
 class Post extends Model
 {
-    protected $fillable = ['slug', 'title', 'content', 'excerpt'];
+    protected $fillable = [
+        'slug',
+        'title',
+        'content',
+        'excerpt'
+    ];
+
+    const DATE_FORMAT = 'd-m-Y H-i';
```

Comment:
Please move it to some config variable - `date_time_format`.

Reply:
Done